src/Parser.js

```diff
@@ -399,6 +400,29 @@ export default class Parser {
                 }
                 // Put everything into an ordgroup as the superscript
                 superscript = {type: "ordgroup", mode: this.mode, body: primes};
+            } else if (uSubsAndSups[lex.text]) {
+                // A Unicode subscript or superscript character.
+                // We treat these similarly to the unicode-math package.
+                // So we render a string of Unicode (sub|super)scripts the
+                // same as a (sub|super)script of regular characters.
+                let str = uSubsAndSups[lex.text];
+                const scriptType = unicodeSubRegEx.test(lex.text) ? SUB : SUPER;
```

Comment:
The `SUB`/`SUPER` codes seem a bit overly complicated. How about just an `isSub` boolean?

Reply:
Done.